js/repl/public/index.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
```

Comment:
Agreed! This is just detritus from my demo site. I haven't yet looked into how to share HTML with the main babel site. 😄